python/lsst/ap/association/packageAlerts.py

```diff
@@ -247,7 +249,14 @@ def run(self,
         diffImPhotoCalib = diffIm.getPhotoCalib()
         calexpPhotoCalib = calexp.getPhotoCalib()
         templatePhotoCalib = template.getPhotoCalib()
+
+        n_sources = len(diaSourceCat)
+        self.log.info("Packaging alerts for %d DiaSources.", n_sources)
+        # Log every 10 seconds as proof of liveness.
+        loop_logger = lsst.utils.logging.PeriodicLogger(self.log, interval=10.0, level=logging.DEBUG)
```

Comment:
Using debug here means that the log messages won't turn up for the batch systems to use as proof of liveness. We tend to use VERBOSE log level (which you can get from `self.log.VERBOSE`) for messages that we don't want people to see if they are running locally but we do want batch systems to see (although 10 seconds seems quite fast).

Reply:
Indeed, the target client is Prompt Processing (where ideally _any single task_ should run in under 10 seconds, and thus never trigger this logger).